src/data/markdown/docs/01 guides/02 Using k6/04 Thresholds.md

```diff
@@ -9,56 +9,55 @@ Thresholds are a pass/fail criteria used to specify the performance expectations
 
 Example expectations (Thresholds):
 
-- System doesn't produce more than 1% errors
+- System doesn't produce more than 1% errors.
 - Response time for 95% of requests should be below 200ms.
 - Response time for 99% of requests should be below 400ms.
 - Specific endpoint must always respond within 300ms.
+- Any conditions on any [Custom metric](/using-k6/metrics#custom-metrics).
 
 Thresholds analyze the performance metrics and determine the final test result (pass/fail).
 Thresholds are a essential for [load-testing automation](/testing-guides/automated-performance-testing).
 
-Here is a sample script that specifies two thresholds, one using a custom [Rate metric](/javascript-api/k6-metrics/rate), and one using a standard `http_req_duration` metric.
+Here is a sample script that specifies two thresholds, one evaluating the rate of http errors (`http_req_failed` metric) and one using the 95 percentile of all the response durations (the `http_req_duration` metric).
 
 <CodeGroup labels={["threshold.js"]} lineNumbers={[true]}>
 
 ```javascript
 import http from 'k6/http';
-import { Rate } from 'k6/metrics';
-
-const myFailRate = new Rate('failed requests');
 
 export let options = {
   thresholds: {
-    'failed requests': ['rate<0.1'], // threshold on a custom metric
-    http_req_duration: ['p(95)<500'], // threshold on a standard metric
+    http_req_failed: ['rate<0.01'],   // http errors should be less than 1% 
+    http_req_duration: ['p(95)<500'], // 95% of requests should be below 200ms
   },
 };
 
 export default function () {
-  let res = http.get('https://test-api.k6.io/public/crocodiles/1/');
-  myFailRate.add(res.status !== 200);
+  http.get('https://test-api.k6.io/public/crocodiles/1/');
```

Comment:
maybe an explanation on the fact that it checks the status code and that is configured will be nice here. Possibly with an example expecting a 404 as there isn't a crocodile and then creating one, and expecting 200 and then deleting it again?